test/NuGet.Core.Tests/NuGet.Protocol.VisualStudio.Tests/PowershellAutoCompleteResourceTests.cs

```diff
@@ -16,15 +21,54 @@ public class PowershellAutoCompleteResourceTests
         public PowershellAutoCompleteResourceTests()
         {
             ResponsesDict = new Dictionary<string, string>();
-            ResponsesDict.Add("http://testsource.com/v3/index.json", JsonData.IndexJson);
-            ResponsesDict.Add("https://api-v3search-0.nuget.org/autocomplete?q=elm", JsonData.PsAutoCompleteV3Example);
-            ResponsesDict.Add("https://nuget.org/api/v2/package-ids?partialId=elm", JsonData.PSAutoCompleteV2Example);
+            ResponsesDict.Add(
+                "http://testsource.com/v3/index.json",
```

Comment:
How about http://source.test/v3/index.json instead?  [.test](https://en.wikipedia.org/wiki/.test) is a top-level domain name reserved for non-production use.  http://testsource.com is a real domain name that redirects to http://www.healthcaresource.com.  Although we may not make a network call here, using the .test top-level domain in tests whenever we need bogus URL's seems like a good practice.